lesson1.1/src/lesson1.java

```diff
@@ -0,0 +1,5 @@
+public class lesson1 {
```

Comment:
According to the Java naming conventions (https://www.javatpoint.com/java-naming-conventions), you should rename class (and file) to Lesson1. Shift + F6 shortcut in Intellij idea will help you